index.js

```diff
@@ -0,0 +1,75 @@
+// EXERCISE 1: Selecting Elements
+// TODO: Select the element with the ID 'mainTitle' and log it to the console.
+let mainTitle = document.getElementById('mainTitle');
+console.log(mainTitle);
+// Exercise 2: Changing HTML Content
+// TODO: Select all paragraph elements and change the content of the second paragraph to "This paragraph has been changed."
+let paragraphs = document.getElementsByTagName('p');
+if (paragraphs.length >= 2) {
+    paragraphs[1].textContent = "This paragraph has been changed.";
+}
+
+// Exercise 3: Changing CSS Styles
+// TODO: Select the first list item and change its text color to red
+// TODO: Select the first list item and change its text color to red
+let firstListItem = document.querySelector('li');
+if (firstListItem) {
```

Comment:
Interesting implementation. This approach is called `defensive programming`... do you know why?

- [ ] Justify defensive approach

Reply:
The goal is to prevent the program from crashing. If the value 'li' is null, this helps prevent the program from crashing.